ddl/ddl.go

```diff
@@ -1206,6 +1206,11 @@ func (d *ddl) CreateIndex(ctx context.Context, ti ast.Ident, unique bool, indexN
 		return errors.Trace(err)
 	}
 
+	// Deal with anonymous index.
+	if len(indexName.L) == 0 {
+		indexName = idxColNames[0].Column.Name
```

Comment:
1. Will the index(c1,c2) and index(c1) conflict? Add the test.
2. Is it possible that idxColNames is empty?